task/s2i-angular-nginx/0.1/s2i-angular-nginx.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  annotations:
+    tekton.dev/displayName: s2i angular nginx
+    tekton.dev/categories: Image Build
+    tekton.dev/tags: image, build, angular, nginx
+    tekton.dev/pipelines.minVersion: '0.19'
+  name: s2i-angular-nginx
+  labels:
+    app.kubernetes.io/version: '0.1'
+spec:
+  params:
+    - default: '16'
+      description: The tag of nodejs version
+      name: NODEJS_VERSION
+      type: string
+    - default: .
+      description: The location of the path to run s2i from.
+      name: CONTEXT_PATH
+      type: string
+    - default: 'true'
+      description: >-
+        Verify the TLS on the registry endpoint (for push/pull to a non-TLS
+        registry)
+      name: TLSVERIFY
+      type: string
+    - description: Location of the repo where image has to be pushed
+      name: IMAGE
+      type: string
+    - default: >-
+        quay.io/buildah/stable:latest
+      description: The location of the buildah builder image.
+      name: BUILDER_IMAGE
+      type: string
+    - name: BUILD_COMMAND
+      description: build command used to compile the source of the application
+      default: 'RUN npm run build -- --prod'
```

Comment:
Option "--prod" is deprecated. I would remove it.

https://angular.io/cli/build